packages/default/scss/button/_layout.scss

```diff
@@ -106,15 +116,21 @@
         margin: 0;
         padding: 0;
         border-width: 0;
+        box-sizing: border-box;
         color: inherit;
         background-color: transparent;
         list-style: none;
         outline: 0;
+        font-family: $button-font-family;
+        font-size: $button-font-size;
+        line-height: $button-line-height;
```

Comment:
the button group is a simple container for buttons. it doesn't really need those